user-registration-app/src/environments/environment.ts

```diff
@@ -18,9 +18,10 @@ export const environment = {
     messagingSenderId: "614592542726"
   },
   // hackathonStartTime: new Date(2018, 10,6, 7),
-  liveWebsiteGuardTime: new Date(2019, 9, 31, 21),
+
+  liveWebsiteGuardTime: new Date(2019, 9, 29, 21),
   hackathonStartTime: new Date(),
-  timerStartTime: new Date(2019, 10, 2, 14),
-  hackathonEndTime: new Date(2019, 10, 3, 14),
+  timerStartTime: new Date(2019, 10, 2, 2),
+  hackathonEndTime: new Date(2019, 10, 3, 2),
```

Comment:
I don't quite understand how these dates are working in conjunction with the timer, but the countdown on my copy of the branch is telling me the hackathon starts in 1 day, 7 hours. This puts the hackathon start time at 2am on Saturday, where it should be 2pm instead.

Reply:
very good point

Reply:
Fixed